apps/frontend/src/app/PageTeams/TeamCard.tsx

```diff
@@ -62,49 +63,44 @@ export default function TeamCard({
         flexDirection: 'column',
       }}
     >
-      <CardActionArea
-        onClick={onClick}
+      <Box
         sx={{
+          height: '100%',
+          p: 1,
           display: 'flex',
           flexDirection: 'column',
           gap: 1,
-          alignItems: 'stretch',
-          flexGrow: 1,
         }}
       >
-        <Box
-          sx={{
-            height: '100%',
-            p: 1,
-            display: 'flex',
-            flexDirection: 'column',
-            gap: 1,
-          }}
-        >
-          <Typography sx={{ display: 'flex', gap: 1 }}>
-            <span>{name}</span>{' '}
-            <BootstrapTooltip title={<Typography>{description}</Typography>}>
-              <InfoIcon />
-            </BootstrapTooltip>
-          </Typography>
+        <Typography sx={{ display: 'flex', gap: 1 }}>
+          <span>{name}</span>{' '}
+          <BootstrapTooltip title={<Typography>{description}</Typography>}>
+            <InfoIcon />
+          </BootstrapTooltip>
+        </Typography>
 
-          <Box sx={{ marginTop: 'auto' }}>
-            <Grid container columns={4} spacing={1}>
-              {range(0, 3).map((i) => (
-                <Grid key={i} item xs={1} height="100%">
-                  {teamCharIds[i] ? (
+        <Box sx={{ marginTop: 'auto' }}>
+          <Grid container columns={4} spacing={1}>
+            {range(0, 3).map((i) => (
+              <Grid key={i} item xs={1} height="100%">
+                {teamCharIds[i] ? (
+                  <CardActionArea
+                    onClick={() =>
+                      onClick(database.teamChars.get(teamCharIds[i]).key)
+                    }
+                  >
                     <CharacterCardPico
                       characterKey={database.teamChars.get(teamCharIds[i]).key}
                     />
-                  ) : (
-                    <BlankCharacterCardPico index={i} />
-                  )}
-                </Grid>
-              ))}
-            </Grid>
-          </Box>
+                  </CardActionArea>
+                ) : (
+                  <BlankCharacterCardPico index={i} />
```

Comment:
What do you think about adding an corresponding `onClick` on the blank char cards to go to the team as well? just default it to the 1st character.